### ISSUE TITLE
[AutoRot] Weird Targeting on Tanks

### ISSUE THREAD

**Issue by Burle17** [PunishXIV/WrathCombo#183]
ok i level at the moment GNB so i do HUNTS in Othard and have to kill Rhino beetles and short before it dies it just changed target to a 40 yalme far away exergon

![image](https://github.com/user-attachments/assets/ce0918e7-bcf7-417f-a5cf-90c9baca91c6)

<details><summary>here is the debug:</summary>

```
START DEBUG LOG

Plugin Version: 1.0.0.4
Installation Repo: https://love.puni.sh/ment.json

Conflicting Plugins: 0
Current Job: Revolverklinge / Gunbreaker / REV
Current Job Index: 37
Current Job Level: 65

Current Zone: Rubinsee
Current Party Size: 1

START ENABLED FEATURES
7003 - GNB_ST_Advanced
7004 - GNB_ST_RangedUptime
7005 - GNB_ST_Continuation
7007 - GNB_ST_Advanced_CooldownsGroup
7008 - GNB_ST_NoMercy
7009 - GNB_ST_BlastingZone
7010 - GNB_ST_BowShock
7011 - GNB_ST_Bloodfest
7012 - GNB_ST_SonicBreak
7014 - GNB_ST_Reign
7015 - GNB_ST_BurstStrike
7016 - GNB_ST_Gnashing
7017 - GNB_ST_DoubleDown
7018 - GNB_ST_Overcap
7019 - GNB_ST_Mitigation
7020 - GNB_ST_HOC
7021 - GNB_ST_GreatNebula
7022 - GNB_ST_Superbolide
7023 - GNB_AuroraProtection
7200 - GNB_AoE_Advanced
7201 - GNB_AoE_NoMercy
7202 - GNB_AoE_DangerZone
7203 - GNB_AoE_BowShock
7204 - GNB_AoE_Bloodfest
7205 - GNB_AoE_SonicBreak
7206 - GNB_AoE_DoubleDown
7207 - GNB_AoE_Reign
7208 - GNB_AoE_FatedCircle
7209 - GNB_AoE_Overcap
7211 - GNB_AoE_BSOvercap
7212 - GNB_AoE_noFatedCircle
7213 - GNB_AoE_HOC
7214 - GNB_AoE_GreatNebula
7215 - GNB_AoE_Superbolide
7216 - GNB_AoE_Mitigation
100000 - ALL_Tank_Interrupt
100001 - ALL_Tank_Reprisal
100020 - ALL_Caster_Addle
100030 - ALL_Melee_Feint
100031 - ALL_Melee_TrueNorth
100040 - ALL_Ranged_Mitigation
100041 - ALL_Ranged_Interrupt
100093 - ALL_IslandSanctuary_Sprint
100095 - ALL_Ranged_Menu
100096 - ALL_Melee_Menu
100097 - ALL_Caster_Menu
100099 - ALL_Tank_Menu
END ENABLED FEATURES

START CONFIG SETTINGS
```

</details>

**Comment by Burle17** [comment]
oh i forgot 

the rota stopped of course i think because he want to do rota on that exergon but i cant because way to far away and also i set autorota to attack NEAREST

![image](https://github.com/user-attachments/assets/30417757-2212-47f8-b757-580bcfabab16)

**Comment by Burle17** [comment]
and i can reproduce this it seems that the targeting system always "prefers" Exergons

because normal killing no quest fate or hunt

![image](https://github.com/user-attachments/assets/00f73c41-a762-48fd-8b81-904c8f452f0c)

i targeted the alba with the distanceattack and my chocobo run to the alba and the exergon is on my back not even close to me and the next thing autorota does is change target and hit the water exergon

**Comment by zbee** [comment]
Likely fixed in d67871a or 16e2422, will be in next release for you to re-test

**Comment by Burle17** [comment]
ok
